src/styles/Calculator.css

```diff
@@ -4,13 +4,25 @@
   padding: 0;
 }
 
-.App {
-  width: 35%;
+.calculator-Page {
+  width: 45%;
   height: 730px;
-  margin-left: 33%;
+  margin-left: 38%;
   margin-top: 13%;
 }
 
+.Calculator-container {
+  width: 50%;
+  margin-left: 25%;
+  margin-top: 90px;
+  margin-bottom: 20px;
+}
+
+.Calculator-header {
+  margin-bottom: 5%;
+  font-size: 30px;
+}
+
```

Comment:
- Your app functions well, only that it will be a lot better to match the general requirement even though there is no heavy demands for perfect styling. However, it is beneficial to stay conscious of the fact that your next viewer might be a potential recruiter or an employer who could be fascinated by the interest and commitment you display on small tasks. :100: 
- [ ] At the moment, there is isn't too much content on the page to justify the vertical scrolling before using the app, and I guess this why the project requirement specifies displaying both elements ("Let's do some maths" and the calculator) side by side. You could easily use flex to achieve this :hammer_and_wrench: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/230031114-a91ac78e-f787-454f-9de3-e7cafb108f17.png)|![image](https://user-images.githubusercontent.com/99715640/230030298-7bdb536e-1481-4ab9-abcd-699d945ec5bc.png)|

- [ ] Also, on the navigation as shown above, kindly reduce the spacing to make it even on both sides.
- [ ] Lastly, I think removing the `border-bottom` on your logo will also make the app look more user-friendly.